modules/tinymce/src/core/main/ts/mode/Readonly.ts

```diff
@@ -120,13 +120,34 @@ const registerReadOnlyContentFilters = (editor: Editor) => {
 
 const isClickEvent = (e: Event): e is MouseEvent => e.type === 'click';
 
-const isInAnchor = (editor: Editor, target: HTMLElement) => editor.dom.getParent(target, 'a') !== null;
-
-const preventReadOnlyEvents = (editor: Editor, e: Event) => {
-  const target = e.target as HTMLElement;
+/*
+* This function is exported for unit testing purposes only
+*/
+const getHrefOpt = (editor: Editor, elm: SugarElement): Optional<string> => {
```

Comment:
Should we call this `getAnchorHrefOpt` just to clarify it only looks at anchor elements?